### PR TITLE
fix: CE-615

### PR DESCRIPTION
**Pull request by barrfalk**
# Description

Updated variable names
Users can now set the "Removed By" user of a piece of equipment, as part of an update.

# How Has This Been Tested?

- [x] Create an equipment record without a "Removed" user/date
- [x] Update the equipment record to set the "removed by" user

## Further comments

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-compliance-enforcement-342-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-compliance-enforcement-342-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge-main.yml)

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-compliance-enforcement-342-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-compliance-enforcement-342-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by Scarlett-Truong** [comment]
Everything function as expected 😃
Only 2 small css that I found:  

- errorMessage for `Equipment Type`, `Remove By`, `Remove Date` are bigger than `Address`, `X Y coordinates`
- their label are kinda shifted down, not center aligned when the errorMessgae shows up
Screen Shot 2024-04-19 at 1 40 07 PM

**Comment by afwilcox** [comment]
@Scarlett-Truong @barrfalk  - I thought as per Product Owner direction this morning we didn't want to be holding up approvals of code for these kind of small UI issues?

The process would be to raise a bug rather than waiting for the item to be fixed.

**Comment by Scarlett-Truong** [comment]
@afwilcox then this PR is good to me. Just a question who want to create a ticket for the UI bug? 😃

**Review by Scarlett-Truong — APPROVED** [review]

**Review by afwilcox — APPROVED** [review]

**Pull request merged**
